sematic/scheduling/job_scheduler.py

```diff
@@ -274,3 +277,41 @@ def _schedule_resolution_job(
         max_parallelism=max_parallelism,
         rerun_from=rerun_from,
     )
+
+
+@unique
+class JobCleaningStateChange(Enum):
+    DELETED = "DELETED"
+    UNMODIFIED = "UNMODIFIED"
+    DELETION_ERROR = "DELETION_ERROR"
+    FORCE_DELETED = "FORCE_DELETED"
+
+
+def clean_jobs(jobs: List[Job], force: bool) -> List[JobCleaningStateChange]:
+    changes = []
+    for job in jobs:
+        if job.state in KubernetesJobState.terminal_states():
+            changes.append(JobCleaningStateChange.UNMODIFIED)
+            continue
```

Comment:
Please also log unmodified jobs, in order to be explicit.

Reply:
done